data/json/items/tool/deployable.json

```diff
@@ -331,5 +331,41 @@
     "symbol": "T",
     "color": "red",
     "flags": [ "DURABLE_MELEE" ]
+  },
+  {
+    "id": "beverly_shear",
+    "type": "TOOL",
+    "name": { "str": "Beverly shear" },
+    "description": "A massive bench-mounted device with a short guillotine blade mounted to a long lever.  Designed for making straight or curved cuts through sheet metal up to 3 millimeters thick.",
+    "//": "Based on the Beverly B-2 Throatless Shear, 10 gauge capacity - as was furniture - values based on real world Beverly B-2 Throatless Shear",
+    "weight": "15875 g",
+    "volume": "50 L",
+    "longest_side": "100 cm",
+    "price": 10000,
+    "price_postapoc": 1500,
+    "to_hit": -5,
+    "bashing": 8,
+    "material": [ "steel" ],
+    "symbol": "7",
+    "color": "yellow_red",
+    "use_action": { "type": "deploy_furn", "furn_type": "f_beverly_shear" }
+  },
+  {
+    "id": "manual_tire_changer",
+    "type": "TOOL",
+    "name": { "str": "manual tire changer" },
+    "description": "This floor mounted machine contains all the levers, clamps and mounts required to remove or fit a tire, plus an integrated foot pump to inflate it.",
+    "//": "Based on the Redline TC500M Manual Tire Changer",
+    "weight": "162000 g",
+    "volume": "500 L",
```

Comment:
Same reasoning here. I haven't found exact measurements but 100l seems to be significantly more realistic, maybe even less.
Height is about 150cm with the bar and the rest is relatively thin material.

Reply:
Same as above - I was more thinking about general item bulkiness. I have also run into issue with finding exact dimensions. Shipping package would be above 750L and item is in general bulky and unwieldy, therefore I settled for 500L as cargo part capacity.

Nonetheless, since volume value should depend on item itself (as mentioned on Discord), and shipping package would be mostly air and packing material, I think your suggestion of 100L is reasonable. Honestly, I think that real volume is even lower, but  the empty spaces between would still be very hard to use and I am not convinced we should lower value even further.